### ISSUE TITLE
A11y_.NETWPF_EditExaminer_ImmediateWindow_HighContrast: At high contrast black and also in high contrast white, focus is not visible in "No exception! Warning Box"

### ISSUE THREAD

**Issue by v-navesingh** [dotnet/wpf#5037]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### **GitHubTags**
#A11ySev2;#A11yMAS;#A11yTCS;#A11yWCAG;#DesktopApp;#DesktopApp;#Win10;#BM-.NetCore(WPF)-Win32-July2021;.NET Core;#Contrast;#WCAG2.4.7;#FocusVisible

### **Environment details:**

Application Name: .NET Core WPF
Microsoft .NET Core SDK Version 6.0.100-rc.1.21380.2
Windows Version: Windows10

### **Repro Steps**

1. Launch VS 2019 Int Preview
2. Tab and Navigate to Sample Applications and right click on the Editing Examiner Demo and click on build.
3..Then right click on Editing Examiner Demo then Click on Debug-->Start New Instance. 
4..Edit Examiner screen should open.
5.Tab and navigate till Immediate Window Control.
6. Tab and Navigate to "No Exception!" control below immediate Window control
7. Turn on High Contrast both High contrast black and high contrast white mode and bring the focus over the control.

### **Actual Result:**
At high contrast black and also in high contrast white, focus is not visible in "No exception! Warning Box"

### **Expected result:**
At high contrast black and also in high contrast white, focus should be visible in "No exception! Warning Box"

### **User Impact**:
Low Vision users may find difficulty in understanding where the focus currently is.

**Comment by v-navesingh** [comment]
> @v-navesingh please attach a screenshot so the team can understand what the problem is.
**### when Focus is over "No Exception! Warning Box":**
![when focus](https://user-images.githubusercontent.com/87067846/131498036-634a7d16-b647-44fd-bc2a-4ca1d15dbb8b.jpg)

**### when Focus is somewhere else:**
![when no focus](https://user-images.githubusercontent.com/87067846/131498163-a16c7216-49a7-46e4-9356-cb3ca72fd736.jpg)

In both cases unable to observe the changes if the focus is received over the control or not.

**Comment by ayushverma-ms** [comment]
PR for the Fix - https://github.com/microsoft/WPF-Samples/pull/402, https://github.com/microsoft/WPF-Samples/pull/400
